portfolio/src/main/webapp/index.html

```diff
@@ -29,6 +29,16 @@ <h1> Hello!</h1>
                 <button onclick="addRandomFact()"> Tell me more </button>
                 <div id="fact-container"></div>
             </div>
+            <div class="subcontent" id="comments">
+                <p>Leave me a comment!</p>
+                <form name="2" action="/comments" method="POST">
+                    <input type="text" name="commenter" placeholder="Name (Optional)" size = "20" maxlength="20">
+                    <input type="text" name="comment-message" placeholder="Comment" size = "100" maxlength="400" required> 
+                    <input type="hidden" name="parent-comment" value="0"> 
+                    <input type="submit" value="Comment">
```

Comment:
Oh did you know that having an input form lets you post by just clicking enter on the keyboard without clicking any submit button? This also works btw! :)

Reply:
I did not know that, good to know, thankyou!